lib/visitor/strategy/default_strategy.dart

```diff
@@ -184,7 +210,24 @@ class DefaultStrategy implements IVisitor {
         visitor.anonymousId = null;
       }
     } else {
-      Flagship.logger(Level.ALL, "unAuthenticateVisitor method will be ignored in Bucketing configuration");
+      Flagship.logger(Level.ALL,
+          "unAuthenticateVisitor method will be ignored in Bucketing configuration");
+    }
+  }
+
+  @override
+  void onExposure(Modification pModification) {
+    var callback =
+        Flagship.sharedInstance().getConfiguration()?.onVisitorExposed;
+    if (callback != null) {
+      callback(
```

Comment:
a shorter version would be:
```dart
Flagship.sharedInstance().getConfiguration()?.onVisitorExposed?.(VisitorExposed(
              visitor.visitorId, visitor.anonymousId, visitor.getContext()),
          ExposedFlag(
              pModification.key,
              pModification.value,
              pModification.defaultValue,
              FlagMetadata.withMap(pModification.toJsonInformation())));
```

Reply:
Yes why not because we have a null safety . but if we do this w'll have to add "**call**" after **onVisitorExposed**. 

`    Flagship.sharedInstance().getConfiguration()?.onVisitorExposed?.call(
        VisitorExposed(
            visitor.visitorId, visitor.anonymousId, visitor.getContext()),
        ExposedFlag(
            pModification.key,
            pModification.value,
            pModification.defaultValue,
            FlagMetadata.withMap(pModification.toJsonInformation())));`

Reply:
```
Flagship.sharedInstance().getConfiguration()?.onVisitorExposed?.call(
        VisitorExposed(
            visitor.visitorId, visitor.anonymousId, visitor.getContext()),
        ExposedFlag(
            pModification.key,
            pModification.value,
            pModification.defaultValue,
            FlagMetadata.withMap(pModification.toJsonInformation())));
```